x/gov/client/utils/query.go

```diff
@@ -22,6 +22,16 @@ type Proposer struct {
 	Proposer   string `json:"proposer"`
 }
 
+// NewProposer returns a new Proposer given id and proposer
+func NewProposer(proposalID uint64, proposer string) Proposer {
+	return Proposer{proposalID, proposer}
+}
+
+func (p Proposer) String() string {
+	return fmt.Sprintf(`ProposalID: %d
```

Comment:
why not just a single line w/o backticks? Also, I think there may be an extra space?

Reply:
`fmt.Sprintf("Proposal w/ ID %d was proposed by %s", propID, prop)` 👍